libnymea/hardware/electricity.h

```diff
@@ -0,0 +1,84 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+*
+* Copyright 2013 - 2022, nymea GmbH
+* Contact: contact@nymea.io
+*
+* This file is part of nymea.
+* This project including source code and documentation is protected by
+* copyright law, and remains the property of nymea GmbH. All rights, including
+* reproduction, publication, editing and translation, are reserved. The use of
+* this project is subject to the terms of a license agreement to be concluded
+* with nymea GmbH in accordance with the terms of use of nymea GmbH, available
+* under https://nymea.io/license
+*
+* GNU Lesser General Public License Usage
+* Alternatively, this project may be redistributed and/or modified under the
+* terms of the GNU Lesser General Public License as published by the Free
+* Software Foundation; version 3. This project is distributed in the hope that
+* it will be useful, but WITHOUT ANY WARRANTY; without even the implied
+* warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+* Lesser General Public License for more details.
+*
+* You should have received a copy of the GNU Lesser General Public License
+* along with this project. If not, see <https://www.gnu.org/licenses/>.
+*
+* For any further details and any questions please contact us under
+* contact@nymea.io or see our FAQ/Licensing Information on
+* https://nymea.io/license/faq
+*
+* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#ifndef ELECTRICITY_H
+#define ELECTRICITY_H
+
+#include "libnymea.h"
+
+#include <QString>
+#include <qobjectdefs.h>
+
+class LIBNYMEA_EXPORT Electricity
+{
+    Q_GADGET
+public:
+    enum Phase {
+        PhaseUnknown = 0x00,
```

Comment:
Probably more correct would be PhaseNone = 0x00, and if a unknown is really needed (is it?) then perhaps make that 0xFF

Reply:
hmm, the `0x00` is actually only required if you hove no idea how many phases exist. There is no case where you work with `None` phases, so it exists only for initializing a variable like in the `convertPhasesFromString()`.
I think initializing flags with 0 as "invalid/unknown/none" makes sense for me in this case...